components/create_comment/create_comment.jsx

```diff
@@ -192,6 +192,26 @@ export default class CreateComment extends React.PureComponent {
         intl: intlShape.isRequired,
     };
 
+    static getDerivedStateFromProps(props, state) {
+        let updatedState = {
+            createPostErrorId: props.createPostErrorId,
+            rootId: props.rootId,
+            messageInHistory: props.messageInHistory,
+        };
+
+        const rootChanged = props.rootId !== state.rootId;
+        const messageInHistoryChanged = props.messageInHistory !== state.messageInHistory;
+        if (rootChanged || messageInHistoryChanged) {
+            updatedState = {...updatedState, draft: {...props.draft, uploadsInProgress: rootChanged ? [] : props.draft.uploadsInProgress}};
+        }
+
+        if (props.createPostErrorId === 'api.post.create_post.root_id.app_error' && props.createPostErrorId !== state.createPostErrorId) {
+            updatedState = {...updatedState, showPostDeletedModal: true};
```

Comment:
I think the state value `showPostDeletedModal ` should be consumed in `componentDidUpdate` to call the function `this.showPostDeletedModal();` to show the modal

Reply:
`this.showPostDeletedModal` calls `setState` to show the modal, so I think it makes sense for this to be done here.